content/en/nearme/demo_videos/videos.md

```diff
@@ -37,3 +37,15 @@ In this video, we look through the different ways to observe your Near Me Virtua
 In this video, we look through the ways your can customize and improve your Near Me Virtual Agent.
 
    <iframe width="560" height="315" src="https://www.youtube.com/embed/sWhigyIh3Gw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>
+
+## Authorize Seasalt.ai on Google Business Profile
+
+In this video, we show the process of authorizing Seasalt.ai to access your virtual agent through Google Business Profile.
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/Y8x0ge51R4U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Can you please copy the style from other widget? At least the border radius is missing ha

Reply:
Oops, yep, missed that!